presto-product-tests/conf/docker/common/standard.yml

```diff
@@ -10,7 +10,7 @@ services:
   hadoop-master:
     extends:
       service: java-8-base
-    image: ${HADOOP_MASTER_IMAGE}
+    image: '${HIVE_IMAGE_PREFIX}:${DOCKER_IMAGES_VERSION}'
```

Comment:
I'm not going to fight over it too much, but have two issues
* HADOOP doesn't say there's HIVE there, HIVE implies there's HADOOP
* BASE_IMAGE suggests there's an extending image, there's not. Maybe BASE_IMAGE_NAME? would be a compromise?

Reply:
It is typical that Hadoop distribution comes with Hive.

> BASE_IMAGE suggests there's an extending image

And there is. The one with suffix `-kerberized`. Possibly there might be more, with different suffixes in the future (oh, god...  please no!).

Reply:
I know you can find arguments for it, still like the other better because it's just straightforward. But fine, I don't feel strong about it. Changing to yours.

Reply:
> HIVE implies there's Hadoop

For me, it doesn't. This could be lonely metastore that has no hdfs (and is configured to use s3 as storage).

BASE_IMAGE vs BASE_IMAGE_NAME -- no big difference to me. Since it's a VARCHAR ( ;) ), it must be some kind of NAME anyway